app/src/main/AndroidManifest.xml

```diff
@@ -132,22 +131,6 @@
         <category android:name="android.intent.category.DEFAULT" />
       </intent-filter>
     </activity>
-    <activity
-      android:name=".localFileTransfer.LocalFileTransferActivity"
-      android:label="Send to nearby device"
```

Comment:
Ah, this was unstranslatable and the only thing that made the title of the screen change, right now it just says "Kiwix", Can you add string resources "Send Files"/"Receive FIles" and use them depending on the context?